scripts/version-linter.py

```diff
@@ -24,26 +24,58 @@
 import sys
 import re
 import sexpdata
+import requests
 
 exit_code=0
 
 def set_error():
   global exit_code
   exit_code=1
 
-def branch_commit(branch):
+def branch_commits(branch,n):
   print ('Retrieving', branch, 'head commit...')
-  result=subprocess.run(['git','log','-n','1','--format="%h"','--abbrev=7','--no-merges',f'{branch}'],
+  result=subprocess.run(['git','log','-n',str(n),'--format="%h"','--abbrev=7','--no-merges',f'{branch}'],
```

Comment:
I think this is the opposite of what we want. We really want the last n 'first-parent' commits, each of which represents a merge to the target branch from a PR. Consider instead
```
git log -n N --format="%h" --abbrev=7 --first-parent BRANCH
```